Dockerfile

```diff
@@ -584,6 +597,15 @@ RUN \
   cd opencore-amr-* && ./configure --enable-static --disable-shared && \
   make -j$(nproc) install
 
+# LIBSSH_STATIC=1 is REQUIRED to link statically against libssh.a so add to pkg-config file
```

Comment:
This seems to be a pkgconfig template at least https://git.libssh.org/projects/libssh.git/tree/libssh.pc.cmake but maybe does not work with static?

Reply:
exactly

Reply:
And doesn't look like alpine had any nice tricks https://git.alpinelinux.org/aports/tree/community/libssh/APKBUILD

Have you seen that they explicitly don't want to support it? thinking maybe worth reporting upstream

But i think your workaround is good

Reply:
yes i looked everywhere including alpine for references

and i havn't seen anything about not supporting whatever - libssh is 'deprecated' in general but ffmpeg uses libssh version 1 until proven otherwise (would be nice if they switched to libssh2)

i'm not sure my answer is what you hoped for...

Reply:
maybe that pc file could be different and still work but i couldn't spend ages on non-essential, its not part of the end binaries and really just helsp me avoid build errors by adding what pkgconf wants

Reply:
No problem and nice workaround it think